src/frontend/components/Chat/SharedChatComponents/ChatConversationDisplayer/index.tsx

```diff
@@ -0,0 +1,56 @@
+import { useMessagesState } from 'data/stores/useMessagesStore';
+import { Box, Spinner } from 'grommet';
+import React, { useEffect, useRef, useState } from 'react';
+import { ChatMessageItem } from '../ChatMessageItem';
+
+export const ChatConversationDisplayer = () => {
+  const { messages } = useMessagesState();
+  const [loading, setLoading] = useState(true);
+  const scrollableContainerRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setLoading(false);
+      scrollableContainerRef.current!.lastElementChild?.scrollIntoView();
+    }, 3000);
+  }, []);
```

Comment:
I don't know if this part is subject to change but I'm not comfortable with the idea to arbitrarily wait for 3 seconds. It's more a hack than a real solution IMHO